### PR TITLE
Completed javascript-if - Lisa Nguyen

### PR DESCRIPTION
**Pull request by lisa-t-nguyen**
image

### CODE REVIEW

File: javascript-if/main.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  }
+}
+
+function startsWithJ(string) {
+  if (string.toLowerCase().charAt(0) === 'j') {
+    return true;
+  } else {
+    return false;
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:32]: `startsWithJ` should only return `true` if `string` starts with a _capital_ `'J'`:

---

image

File: javascript-if/main.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+function isOldEnoughToDrinkAndDrive(person) {
+  if (person.age) {
+    return false;
+  }
+}
```
Comment (TimelikeClosure) [inline javascript-if/main.js:54]: A newborn should also trigger a `false` from `isOldEnoughToDrinkAndDrive`.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-if/main.js (hunk 1 above)
Comment (lisa-t-nguyen) [reply javascript-if/main.js:32]: Thank you! I missed that!

**Comment by lisa-t-nguyen** [comment]
image

Here is the updated screenshot, I'm not sure how to recognize months/weeks as age for the person object...

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]
> Here is the updated screenshot, I'm not sure how to recognize months/weeks as age for the person object...

@lisa-t-nguyen By "newborn" I mean `age` is `0`.

---

image

**Comment by lisa-t-nguyen** [comment]
Thank you! I think I was being too specific there

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**